### PR TITLE
Wizard

### PR DESCRIPTION
**Pull request by Coornail**
Fixes https://github.com/mdyd-dev/pos-cli/issues/382

### CODE REVIEW

File: bin/pos-cli-init.js [hunk 1]
```diff
@@ -1,32 +1,77 @@
...
-      .clone('.')
-      .then(() => {
-        report('Init', { extras: [{ key: 'status', value: 'Success' }, { key: 'hasBranch', value: !!params.branch }] });
-        logger.Success('Directory structure sucessfully created.');
-      })
-      .catch(error => {
-        report('Init', { extras: [{ key: 'status', value: 'Error' }, { key: 'hasBranch', value: !!params.branch }] });
-        logger.Error(`Cloning failed. Reason: ${error.message}`);
-      });
+  .option('--wizard', 'Start repo wizzard')
```
Comment (pavelloz) [inline bin/pos-cli-init.js:48]:
```suggestion
  .option('--wizard', 'Start repo wizard')
```

File: bin/pos-cli-init.js [hunk 2]
```diff
@@ -1,32 +1,77 @@
...
+  degit = require('degit'),
+  inquirer = require('inquirer');
 
 const logger = require('../lib/logger'),
   report = require('../lib/logger/report');
 
+const repos = {
+  empty: 'mdyd-dev/directory-structure',
+  'Hello world': 'mdyd-dev/hello-world',
+  'Todo app': 'mdyd-dev/todo-app.git',
```
Comment (pavelloz) [inline bin/pos-cli-init.js:13]:
```suggestion
  'Todo app': 'mdyd-dev/todo-app',
  'Product Marketplace Template': 'mdyd-dev/product-marketplace-template'
```

**Comment by pavelloz** [comment]
Its not a breaking change right?

`pos-cli init`
`pos-cli init --url x --branch y`

will work the same without triggering wizard am i correct?

**Comment by Coornail** [comment]
That is correct, yes.

**Pull request merged**